tests/Microsoft.Identity.Test.Unit/CoreTests/CollectionHelperTests.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Identity.Test.Unit.CoreTests
     [TestClass]
     public class CollectionHelperTests
     {
-#if NETCOREAPP
+#if NET6_WIN
```

Comment:
This test is no longer functional on netstandard due to missing types needed for the test